report/report.tex

```diff
@@ -108,19 +110,24 @@ \section{Problem Formulation}
 %------------------------------------------------
 \section{Methods}
 
-Maecenas sed ultricies felis. Sed imperdiet dictum arcu a egestas.
-\begin{itemize}
-	\item Donec dolor arcu, rutrum id molestie in, viverra sed diam
-	\item Curabitur feugiat
-	\item turpis sed auctor facilisis
-	\item arcu eros accumsan lorem, at posuere mi diam sit amet tortor
-	\item Fusce fermentum, mi sit amet euismod rutrum
-	\item sem lorem molestie diam, iaculis aliquet sapien tortor non nisi
-	\item Pellentesque bibendum pretium aliquet
-\end{itemize}
-\blindtext % Dummy text
+\subsection{Data exploration}
+
+With the cleaned up dataset and the research question in mind, columns with relevance are explored. To explore, we plot different columns of interest to identify any patterns in the data. We plot the distribution between reviews with votes and no votes.
```

Comment:
To explore, we plot different columns of interest to identify any patterns in the data in connection to review helpfulness. 

Tænker det er godt lige at få det linked til vores hovedspørgsmål :D

Reply:
Done :DD